internal/host/statemachine.go

```diff
@@ -506,8 +506,12 @@ func NewHostStateMachine(th *transitionHandler) stateswitch.StateMachine {
 
 	var requiredInputFieldsExist = stateswitch.And(If(IsMachineCidrDefined))
 
-	var isSufficientForInstall = stateswitch.And(If(HasMemoryForRole), If(HasCPUCoresForRole), If(BelongsToMachineCidr), If(IsHostnameUnique), If(IsHostnameValid), If(IsAPIVipConnected), If(BelongsToMajorityGroup),
-		If(AreOcsRequirementsSatisfied), If(AreLsoRequirementsSatisfied), If(AreCnvRequirementsSatisfied), If(SufficientOrUnknownInstallationDiskSpeed), If(SucessfullOrUnknownContainerImagesAvailability), If(HasSufficientNetworkLatencyRequirementForRole), If(HasSufficientPacketLossRequirementForRole))
+	var operatorRequirementsSatisfied = stateswitch.And(If(AreOcsRequirementsSatisfied), If(AreLsoRequirementsSatisfied), If(AreCnvRequirementsSatisfied))
```

Comment:
Where are these conditions implemented?

Reply:
These conditions are implement in OLM operators plugins. For example, for CNV: https://github.com/openshift/assisted-service/blob/f5a57dc029e010df6aaf236797c800ab160d584e/internal/operators/cnv/cnv_operator.go#L77

The result of that check is then taken into account here: https://github.com/openshift/assisted-service/blob/f5a57dc029e010df6aaf236797c800ab160d584e/internal/host/refresh_status_preprocessor.go#L96